tests/css_less/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,52 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`function-each.less 1`] = `
+====================================options=====================================
+parsers: ["less"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+// http://lesscss.org/functions/#list-functions-each
+
+@selectors: blue, green, red;
+
+each(@selectors, {
+  .sel-@{value} {
+    a: b;
+  }
+});
+
+
+// TODO: fix this
+each(@selectors, 
+{
+  .sel-@{value} {
+a: b;
+  }
+});
+
+=====================================output=====================================
+// http://lesscss.org/functions/#list-functions-each
+
+@selectors: blue, green, red;
+
+each(@selectors, {
+  .sel-@{value} {
+    a: b;
+  }
+});
+
+// TODO: fix this
+each(@selectors, 
+{
+  .sel-@{value} {
+a: b;
```

Comment:
Please open issue about that after merge

Reply:
I think it's https://github.com/prettier/prettier/issues/4524, not a new issue.